package.json

```
@@ -34,7 +34,7 @@
   },
   "lint-staged": {
     "**/!(images)/**/*.{ts,js,html,scss,md,mdx,json}": [
-      "yarn affected:lint -- --uncommitted --fix --parallel"
+      "nx affected:lint --uncommitted --fix --parallel"
```

won't this mean it will always use the users globally installed version of `nx` instead of the one living in node_modules?

---

```suggestion
      "yarn affected:lint --uncommitted --fix --parallel 4"
```

Also, I have the feeling `affected:lint --uncommitted` is doing way way more than needed. We should probably rely only on eslint, without affected. But that's a topic for another PR.

---

I assume everyone has 4 threads available btw ^^